besu/src/main/java/org/hyperledger/besu/Runner.java

```diff
@@ -253,7 +253,11 @@ private void waitForServiceToStop(final String serviceName, final SynchronousShu
     try {
       shutdown.await();
     } catch (final InterruptedException e) {
-      LOG.debug("Interrupted while waiting for service " + serviceName + " to stop", e);
+      LOG.atDebug()
```

Comment:
in this case, and in most of the cases below, you can use the more compact form:

```
LOG.debug("Interrupted while waiting for service {} to stop", serviceName, e);
```

that has the same effect and maybe better performance, since the `serviceName.toString()` is only called if DEBUG is enabled, the rule of thumb is to used `atDebug` and similar, when some computation needs to be done to get the string to log, otherwise the compact form is easier to read.